cli/internal/helm/charts/edgeless/constellation-services/values.yaml

```diff
@@ -10,6 +10,14 @@ global:
   # Name of the ConfigMap that holds configs that should not be modified by the user.
   internalCMName: internal-config
 
+# GCP specific configuration
+gcp:
+  deployCSIDriver: false
+
+# Azure specific configuration
+azure:
+  deployCSIDriver: false
```

Comment:
Tried this before, but had some issues where both drivers were deployed .
Seems like something else was wrong tho. And it works now
Thanks for the reminder :+1:

Reply:
This does not work since conditions overwrite tags, e.g. if a condition to deploy a dependency chart is true, any tags are ignored.
So using a global `deployCSIDriver` value would deploy all drivers regardless of CSP.

This is a annoyance of helm.
IMO their whole tag/condition management is a bit confusing and doesnt make a whole lot sense

Reply:
Ugh. That _is_ annoying. Alrighty.